tests/integration/ibc_integration_test.go

```diff
@@ -85,7 +85,7 @@ func TestIBCReflectContract(t *testing.T) {
 	// only writes I see: https://github.com/cosmos/cosmos-sdk/blob/31fdee0228bd6f3e787489c8e4434aabc8facb7d/x/ibc/core/04-channel/keeper/packet.go#L115-L116
 	// commitment is hashed packet: https://github.com/cosmos/cosmos-sdk/blob/31fdee0228bd6f3e787489c8e4434aabc8facb7d/x/ibc/core/04-channel/types/packet.go#L14-L34
 	// how is the relayer supposed to get the original packet data??
-	// eg. ibctransfer doesn't store the packet either: https://github.com/cosmos/cosmos-sdk/blob/master/x/ibc/applications/transfer/keeper/relay.go#L145-L162
+	// eg. ibctransfer doesn't store the packet either: https://github.com/cosmos/ibc-go/blob/main/modules/apps/transfer/keeper/relay.go#L141-L158
```

Comment:
That's not the code this used to link to. It most likely used to be this: https://github.com/cosmos/cosmos-sdk/blob/31fdee0228bd6f3e787489c8e4434aabc8facb7d/x/ibc/applications/transfer/keeper/relay.go#L145-L162
Looks like that code doesn't really exist in that form anymore.
Honestly, this whole comment wall is not that useful anyways, but for now, I'd just replace with the correct link:
```suggestion
	// eg. ibctransfer doesn't store the packet either: https://github.com/cosmos/cosmos-sdk/blob/31fdee0228bd6f3e787489c8e4434aabc8facb7d/x/ibc/applications/transfer/keeper/relay.go#L145-L162
```